lib/ruby_lsp/ruby_lsp_rails/hover.rb

```diff
@@ -83,7 +83,8 @@ def generate_column_content(name)
 
         @response_builder.push(
           model[:columns].map do |name, type|
-            "**#{name}**: #{type}\n"
+            primary_key_suffix = " (PK)" if model[:primary_keys].include?(name)
```

Comment:
Question to everybody: should we be verbose and use `PRIMARY KEY`? I have a tendency to prefer verbose over abbreviations.